### DISCUSSION TITLE
SACD ISO file support

### DISCUSSION THREAD

**Discussion by aljus7** [rern/rAudio#589] [Feature Requests]
Looks like there is already a fork of mpd that supports SACD iso: https://sourceforge.net/projects/mpd.sacddecoder.p/ . It's not a deal breaker for me but would be nice feature to have.

**Comment by klassikmann** [comment]
@aljus7 
I know this fork of mpd by Maxim Anisiutkin, author of the foobar2000-plugin foo_input_sacd (among others). In the past I made several tests with his fork for native SACD-ISO: it worked. 

But we would face a maintainability problem using manisiutkin's fork in rAudio: First, status of the fork is "Inactive". Updates are released from time to time, the last one was from 2021-07-05. In the meantime Max Kellermann released mpd 0.22.10, 0.22.11, 0.23, 0.23.1, 0.23.2, 0.23.3. And second, mpd is well maintained in archlinuxarm. New versions are available a few days after the source release.

**Reply by Jameschiukwok** [reply]
what about making a tutorial for those who want to merge the sacd-iso  to any undated MDP?
then this will be good for those who are interested to using udpated newest mdp while keeping the sacd-iso like me?
thanks

**Comment by rern** [comment]
[Support of SACD ISO images in mpd #644](https://github.com/MusicPlayerDaemon/MPD/issues/644)
